### ISSUE TITLE
calcite-input-time-zone throws error on Chrome

### ISSUE THREAD

**Issue by richiecarmichael** [Esri/calcite-design-system#10895]
### Check existing issues
- [X] I have [checked for existing issues](https://github.com/Esri/calcite-design-system/issues) to avoid duplicates

### Actual Behavior
`calcite-input-time-zone` crashes on initialization on Chrome and Edge. Works fine on FF and Safari.

### Expected Behavior
Component does not crash.

### Reproduction Sample
See below

### Reproduction Steps
To reproduce failure with `DateTimeFormat`.
1. Click [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from)
2. Enter the following code:
```js
const t = new Intl.DateTimeFormat("en", {
  timeZone: "Factory",
  timeZoneName: "shortOffset",
});
```
4. Click Run.
### Reproduction Version
3.0.0

### Regression?
Yes

### Priority impact
impact - p1 - need for current milestone

### Impact
Affects the `TimeZoneLabel` JSSDK widget when used on Chrome or Edge.

### Calcite package
- [X] @esri/calcite-components
- [ ] @esri/calcite-components-react
- [ ] @esri/calcite-design-tokens
- [ ] @esri/calcite-ui-icons
- [ ] @esri/eslint-plugin-calcite-components

### Esri team
ArcGIS Maps SDK for JavaScript

### Reference
[#66469](https://devtopia.esri.com/WebGIS/arcgis-js-api/issues/66471) ValuePicker - Migrate to Calcite 3.0

// @jcfranco

**Comment by nCastle1** [comment]
I'm seeing a similar issue when setting up time zone picker both in the usual way through tsx, and by manually creating and attaching to DOM (testing workflow). I see slightly different behavior where creating with document.createElement and appending also breaks on firefox, but firefox works fine through tsx.

**Comment by jcfranco** [comment]
This seems related to a recent Chrome update where “Factory” is no longer a valid timeZone option in Intl.DateTimeFormat. I’ve filed a [Chromium issue](https://issues.chromium.org/issues/381620359) to track this.

@nCastle1, could you share the console error you’re encountering? I suspect it might be a different issue.

**Comment by nCastle1** [comment]
I investigated further, and I think the error was unrelated. I couldn't reproduce in codepen and narrowed down to an asset loading issue in test environment.

**Comment by hcampos-professional** [comment]
Would it be possible to do a workaround on our side? Unclear when the chrome patch would happen, and it would also require testing environments to adopt the patch or skip related tests until that's done.

**Comment by jcfranco** [comment]
We'll look into a workaround while the Chrome bug is being investigated.

**Comment by jcfranco** [comment]
[Fixed this](https://github.com/jcfranco/timezone-groups/commits/main/) in `timezone-groups` and bumped our dependency. This should be fixed in the next `next` deployment.

**Comment by jcfranco** [comment]
Spotted one more issue. Bouncing back. Should be a quick fix.

**Comment by richiecarmichael** [comment]
When the `input-time-zone` component is in "region" `mode` and assigned a `value` of "Factory" the text "System Default Time" as pictured below.

![image](https://github.com/user-attachments/assets/18924f59-3cdb-4ea6-bc30-38b510f1ef7c)

IMHO, "System Default Time" infers the current device's time zone. For me, in Redlands, this would be "America/Los_Angeles" or "Pacific/Auckland" for mum.

From the little documentation I have read, "Factory" means no value or unassigned.

![image](https://github.com/user-attachments/assets/fac07448-835a-4db3-b800-8a3ec625ee5e)

A better textural representation would be something like "unknown" or "not assigned" etc.

**Comment by tayMAPNeil** [comment]
FYI This is also reproducible with v2.13.2 that is currently on Production. The calcite-input-time-zone input does not render in areas like adding fields, forms, etc.

**Comment by geospatialem** [comment]
> When the `input-time-zone` component is in "region" `mode` and assigned a `value` of "Factory" the text "System Default Time" as pictured below.
> 
> IMHO, "System Default Time" infers the current device's time zone. For me, in Redlands, this would be "America/Los_Angeles" or "Pacific/Auckland" for mum.
> 
> From the little documentation I have read, "Factory" means no value or unassigned.
> 
> A better textural representation would be something like "unknown" or "not assigned" etc.

Richie added #10998 above ☝🏻  to investigate the above.  🤜🏻🤛🏻

 
> FYI This is also reproducible with v2.13.2 that is currently on Production. The calcite-input-time-zone input does not render in MapViewer in areas like adding fields, forms, etc.
> 
> Also reproducible on the developer doc: https://developers.arcgis.com/calcite-design-system/components/input-time-zone/

Followed-up with Taylor via Teams that Calcite has mitigated the issue in the interim via `next`. (Note: We _may_ need to consider patching `2.12.x` if needed).

The fixes above are mitigated with Chrome in `3.0.0-next.51`: https://codepen.io/geospatialem/pen/mybVKjX 🎉
